data/json/furniture_and_terrain/furniture-decorative.json

```
@@ -205,5 +205,299 @@
+    "bash": {
+      "str_min": 6,
+      "str_max": 40,
+      "sound": "smash!",
+      "sound_fail": "whump.",
+      "items": [ { "item": "2x4", "count": [ 1, 2 ] }, { "item": "nail", "charges": [ 2, 4 ] }, { "item": "splinter", "count": 2 } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_flamingo_decor",
+    "name": "flamingo decoration",
+    "symbol": "%",
+    "description": "Classy or trashy?  You decide.",
+    "color": "pink",
+    "move_cost_mod": 1,
+    "coverage": 10,
+    "required_str": -1,
+    "deployed_item": "decorative_flamingo",
+    "examine_action": "deployed_furniture",
+    "flags": [ "TRANSPARENT", "FLAMMABLE_ASH" ],
+    "bash": {
+      "str_min": 5,
+      "str_max": 16,
+      "sound": "thump!",
+      "sound_fail": "whack!",
+      "items": [ { "item": "plastic_chunk", "count": [ 1, 3 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_birdfeeder",
+    "name": "bird feeder",
+    "description": "A small hanging bird feeder with a stake.",
+    "symbol": "T",
+    "color": "light_gray",
+    "move_cost_mod": -1,
+    "required_str": -1,
+    "flags": [ "PLACE_ITEM", "TRANSPARENT" ],
+    "bash": {
+      "str_min": 10,
+      "str_max": 40,
+      "sound": "smash!",
+      "sound_fail": "thump.",
+      "items": [
+        { "item": "splinter", "count": [ 1, 2 ] },
+        { "item": "pipe", "count": [ 1, 2 ] },
+        { "item": "wire", "charges": [ 1, 2 ] }
+      ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_traffic_cone",
+    "name": "traffic cone",
+    "description": "A small traffic cone.",
+    "symbol": "^",
+    "color": "red",
+    "move_cost_mod": 1,
+    "required_str": 1,
+    "flags": [ "TRANSPARENT", "SHORT" ],
+    "bash": {
+      "str_min": 5,
+      "str_max": 40,
+      "sound": "whump!",
+      "sound_fail": "thump.",
+      "items": [ { "item": "plastic_chunk", "count": [ 1, 2 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_taxidermy_bear",
+    "name": "bear taxidermy statue",
+    "description": "A large taxidermied bear.",
+    "symbol": "B",
+    "color": "brown",
+    "move_cost_mod": -1,
+    "required_str": 15,
+    "flags": [ "BASHABLE", "FLAMMABLE_ASH" ],
+    "bash": {
+      "str_min": 8,
+      "str_max": 40,
+      "sound": "Crack!",
+      "sound_fail": "thump.",
+      "items": [
+        { "item": "splinter", "count": [ 1, 2 ] },
+        { "item": "leather", "count": [ 1, 2 ] },
+        { "item": "wire", "charges": [ 1, 1 ] }
+      ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_foodperson_cutout",
+    "name": "foodperson cutout",
+    "symbol": "8",
+    "description": "A large lifesize cutout of FoodPerson.",
+    "color": "white",
+    "move_cost_mod": 2,
+    "coverage": 80,
+    "required_str": 1,
+    "flags": [ "FLAMMABLE_ASH", "ORGANIC", "SIGN" ],
+    "deconstruct": { "items": [ { "item": "paper", "count": 15 } ] },
```

```suggestion
    "deconstruct": { "items": [ { "item": "paper", "charges": 15 } ] },
```

`"count": 15` would drop 750 paper, which is far too much.